render/render.go

```diff
@@ -240,18 +243,137 @@ func (p TextRenderer) Render(w io.Writer, x ast.Expr) (mErr error) {
 		_, mErr = w.Write([]byte("-"))
 	case *ast.TextMath:
 		if _, mErr = w.Write([]byte("$")); mErr != nil {
-			return mErr
+			return
 		}
 		if _, mErr = w.Write([]byte(t.Value)); mErr != nil {
-			return mErr
-		}
-		if _, mErr = w.Write([]byte("$")); mErr != nil {
-			return mErr
+			return
 		}
+		_, mErr = w.Write([]byte("$"))
 	case *ast.TextNBSP, *ast.TextSpace:
 		_, mErr = w.Write([]byte(" "))
+	case *ast.TextAccent:
+		var (
+			accent       accentType
+			accentedRune rune
+		)
+		if accent, mErr = newAccentType(t.Accent); mErr != nil {
+			return
+		}
+		text := t.Value.(*ast.Text).Value
+		if utf8.RuneCountInString(text) != 1 {
+			return fmt.Errorf("renderer - accent text must be a single character")
+		}
+		r, _ := utf8.DecodeRuneInString(text)
+		if accentedRune, mErr = fmtAccent(r, accent); mErr != nil {
+			return
+		}
+		_, mErr = w.Write([]byte(string(accentedRune)))
 	default:
-		return fmt.Errorf("renderer - unhandled ast.Expr type %T, %v", t, t)
+		mErr = fmt.Errorf("renderer - unhandled ast.Expr type %T, %v", t, t)
+	}
+	return
+}
+
+// accentType represents the different LaTeX accents
+type accentType string
```

Comment:
Let's move these to accent.go. It turns out there's a lot of accented characters we might want to eventually support: https://github.com/loopspace/latexToUTF/blob/master/utf.php

Reply:
done!